UPGRADE-API-1.12.md

```diff
@@ -225,3 +225,176 @@ The list of changed contexts:
 - Sylius\Behat\Context\Api\Shop\PromotionContext
 - Sylius\Behat\Context\Api\Shop\RegistrationContext
 - Sylius\Behat\Context\Api\Shop\ShipmentContext
+
+### Creating Request refactored
+
+We removed some methods from `Sylius/Behat/Client/RequestInterface`.
```

Comment:
```suggestion
We removed some methods from `Sylius\Behat\Client\RequestInterface`.
```
just to be consistent with the entire MD file and you can do the same with the rest of the references.